pkg/pipeline/transform/infinispan/cache.go

```diff
@@ -0,0 +1,90 @@
+package infinispan
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"regexp"
+
+	ispnclient "github.com/infinispan/infinispan-operator/pkg/http"
+	"github.com/infinispan/infinispan-operator/pkg/mime"
+	"github.com/netobserv/flowlogs-pipeline/pkg/config"
+	"github.com/sirupsen/logrus"
+
+	"github.com/infinispan/infinispan-operator/pkg/infinispan/client/api"
+	v14 "github.com/infinispan/infinispan-operator/pkg/infinispan/client/v14"
+)
+
+var log = logrus.WithField("component", "transform.Network.Cache")
+var regex = regexp.MustCompile(`[^a-zA-Z0-9 ]+`)
```

Comment:
is there forbidden characters in key that require this replaceAll ?

Reply:
I had issues with my initial attempt either with `.` or `:` when I had standart ips + ports key
I can give another try to get rid of this regex and improve perfs indeed

Reply:
also addressed in https://github.com/netobserv/flowlogs-pipeline/pull/639/commits/335d7370beb104917ab9806b4663b6f40a1f42f6